test/debugInfo.fromasm.clamp

```diff
@@ -47,17 +46,11 @@
     (get_local $0)
    )
   )
+  ;;@ even-opted.cpp:3:0
   (i32.add
-   (if (result i32)
-    ;;@ even-opted.cpp:3:0
-    (tee_local $2
-     (get_local $1)
-    )
-    (i32.rem_s
-     (get_local $0)
-     (get_local $2)
-    )
-    (i32.const 0)
+   (call $i32s-rem
```

Comment:
hmm, this used to be inlined, and now it isn't. if this is still the case after the other changes, that's worth investigating.